kubernetes-services/work-failer/test/helpers/jobs.ts

```diff
@@ -0,0 +1,541 @@
+import request, { Test } from 'supertest';
```

Comment:
Similarly this has a ton of code that shouldn't be needed.

Reply:
Again I had to do this to get TypeScript to cooperate. Will clean all this up with a complete refactoring later.